projects/editor/package.json

```diff
@@ -41,9 +42,9 @@
     "Thomas-Louis Simard <thomas.louis.simard@gmail.com>"
   ],
   "peerDependencies": {
-    "@angular/common": "^17.0.3",
-    "@angular/core": "^17.0.3",
-    "monaco-editor": "^0.44.0"
+    "@angular/common": "^18.0.1",
+    "@angular/core": "^17.0.9",
```

Comment:
This should be `^18.0.1` as well, otherwise the package can't be installed due to a dependency conflict